src/main/java/sg/ncl/MainController.java

```diff
@@ -96,10 +97,14 @@
     private static final String NO_PERMISSION_PAGE = "nopermission";
 
     private static final String TEAM_NAME = "teamName";
+    private static final String TEAM_ID = "teamId";
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "teamId" 5 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1192)